javascript/builtins/String.json

```diff
@@ -1130,7 +1130,8 @@
                 "version_added": true
               },
               "edge": {
-                "version_added": true
+                "version_added": true,
+                "version_removed": "14"
```

Comment:
we did not remove the length property from Strings

Reply:
This (and possible other) error(s) appears to come from our [exclude constants from API list heuristic](https://github.com/GoogleChromeLabs/confluence/blob/e10a30d8e346c0836952d7632bc27d7f7756c36c/lib/web_catalog/api_extractor.es6.js#L680). `Object.getOwnPropertyDescriptor(String.prototype, 'length').value` in Edge returns `0`, not `undefined`.

Reply:
@foolip and I are considering combining the existing heuristic with a `CONST_CASE` name check to get a better guess at "constants that can be ignored in method+property counts".